back/engines/free/email_campaigns/app/views/email_campaigns/comment_on_your_comment_mailer/campaign_mail.mjml

```diff
@@ -1,4 +1,4 @@
-<mj-section padding="25px"text-align="left">
+<mj-section padding="25px" text-align="left">
```

Comment:
This is not valid MJML.

Reply:
Do we need to use your new align_direction var here?